README.md

```diff
@@ -1,24 +1,42 @@
-# README
+![](https://img.shields.io/badge/Kiprotich-kimutai-yellow?labelColor=blue)&nbsp;
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Rails api
 
-Things you may want to cover:
+> This is a rails api
 
-* Ruby version
+## Built With
 
-* System dependencies
+---
 
-* Configuration
+![](https://img.shields.io/badge/Rails-black)&nbsp;&nbsp;&nbsp;![](https://img.shields.io/badge/Ruby-green)&nbsp;&nbsp;&nbsp;![](https://img.shields.io/badge/Devise-Yellow)
 
-* Database creation
+[Live to frontend](https://github.com/Kimutai01/hello-react-front-end)
```

Comment:
- [ ] **Required:** Kindly add all the required sections to the README of the frontend.